packages/auth/src/authClients/index.ts

```diff
@@ -1,18 +1,18 @@
-import type { NetlifyIdentity } from './netlify'
 import type { Auth0, Auth0User } from './auth0'
+import { auth0 } from './auth0'
+import type { Custom } from './custom'
+import { custom } from './custom'
+import type { Firebase } from './firebase'
+import { firebase } from './firebase'
 import type { GoTrue, GoTrueUser } from './goTrue'
+import { goTrue } from './goTrue'
 import type { MagicLink, MagicUser } from './magicLink'
-import type { Firebase } from './firebase'
+import { magicLink } from './magicLink'
+import type { NetlifyIdentity } from './netlify'
```

Comment:
Hmmm... this is unfortunate. Is there a way to specify that "type imports" should be right at the top of the file?

Reply:
Probably, I'll look into it

Reply:
No luck with this?

Reply:
Unfortunately, no. See: https://github.com/benmosher/eslint-plugin-import/issues/645

Reply:
Ok, no worries. It's better than nothing.